src/_includes/layouts/base.njk

```diff
@@ -43,7 +43,7 @@
           <a href="https://twitter.com/v8js" rel="me nofollow">Twitter</a> ·
           <a href="https://github.com/v8/v8.dev/tree/master/{{ page.inputPath }}" rel="nofollow">Edit this page on GitHub</a>
         </nav>
-        <dark-mode-toggle permanent dark="Light Theme" light="Dark Theme"></dark-mode-toggle>
+        <dark-mode-toggle dark="Light Theme" light="Dark Theme"></dark-mode-toggle>
```

Comment:
@mathiasbynens This completely removes storage of the chosen option... I don't think that's what we want, I think we just want external color scheme event to modify the stored option just like the toggle does itself.

Reply:
I think what we want is to actually change this bit in dark-mode-toggle:

https://github.com/GoogleChromeLabs/dark-mode-toggle/blob/master/src/dark-mode-toggle.mjs#L141-L144

Currently, it assumes that if user toggled the switch, then it should always persist its state, but what we want is for external event to change mode too.

Reply:
TL;DR - try to remove `if (...)` there.

Reply:
I worked around this without changing dark-mode-toggle by manually listening to external events for now, so either the toggle or external event can change the theme. @mathiasbynens let me know what you think about the behaviour.

Reply:
I like it! Thanks Ingvar :+1: